AsaApi/main.cpp

```diff
@@ -63,6 +63,67 @@ void OpenConsole()
 	SetConsoleOutputCP(CP_UTF8);
 }
 
+void DeleteOldLogFiles()
```

Comment:
We should probably enable some form of log file name configuration in the long term.